### PR TITLE
Added disallowed-registries

### PR DESCRIPTION
**Pull request by michaelkotelnikov**
Solves - https://github.com/openshift-4-compliance/openshift-4-compliance-automation/issues/135

### CODE REVIEW

File: open-policy-agent/trusted-image-sources/disallowed-registries/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Disallow Registries
+
+The policy makes sure a list of "Allowed Registries" is associated with the `cluster` Image resource. If a registry is not mentioned in the Image resource, images from this registry will not be pulled for pod creation.
```
Comment (galshi) [inline open-policy-agent/trusted-image-sources/disallowed-registries/README.md:3]: What what do you mean by "`cluster`" image resource?

File: open-policy-agent/trusted-image-sources/disallowed-registries/template.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          input.review.object.kind == "Image"
+          input.review.object.apiVersion == "config.openshift.io/v1"
+          missing(input.review.object.spec.registrySources, "allowedRegistries")
+          msg := sprintf("%v object must have spec.registrySources.allowedRegistries configured", [input.review.object.kind])
+        }
+
+        violation[{"msg": msg}] {
+          input.review.object.kind == "Image"
+          input.review.object.apiVersion == "config.openshift.io/v1"
+          count(input.review.object.spec.registrySources.allowedRegistries) == 0
```
Comment (galshi) [inline open-policy-agent/trusted-image-sources/disallowed-registries/template.yaml:42]: Wouldn't 
```
 missing(obj, field) = true {
             obj[field] == ""
         }
```
already catch this?

File: open-policy-agent/trusted-image-sources/disallowed-registries/template.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          msg := sprintf("%v object must have at least one registry configured at spec.registrySources.allowedRegistries", [input.review.object.kind])
+        }
+
+        violation[{"msg": msg}] {
+          input.review.object.kind == "Image"
+          input.review.object.apiVersion == "config.openshift.io/v1"
+          allowedRegistries := { registry | registry := input.review.object.parameters.allowedRegistries[_]}
```
Comment (galshi) [inline open-policy-agent/trusted-image-sources/disallowed-registries/template.yaml:49]: Out of ignorance, why can you pass this in two different parameters to begin with?

File: open-policy-agent/trusted-image-sources/disallowed-registries/README.md (hunk 1 above)
Comment (michaelkotelnikov) [reply open-policy-agent/trusted-image-sources/disallowed-registries/README.md:3]: Will add link to docu.

**Pull request merged**